### ISSUE TITLE
Crash report 7.2: IAE in sanitizeUrl, cause U_IDNA_LABEL_TOO_LONG_ERROR

### ISSUE THREAD

**Issue by mzorz** [wordpress-mobile/WordPress-Android#5832]
related to #5667, but note the same Exception has a different cause

```
Fatal Exception: java.lang.IllegalArgumentException: U_IDNA_LABEL_TOO_LONG_ERROR
       at libcore.icu.NativeIDN.convertImpl(NativeIDN.java)
       at libcore.icu.NativeIDN.convert(NativeIDN.java:39)
       at libcore.icu.NativeIDN.toASCII(NativeIDN.java:21)
       at java.net.IDN.toASCII(IDN.java:64)
       at org.wordpress.android.util.UrlUtils.convertUrlToPunycodeIfNeeded(UrlUtils.java:61)
       at org.wordpress.android.fluxc.network.discovery.SelfHostedEndpointFinder.sanitizeSiteUrl(SelfHostedEndpointFinder.java:347)
       at org.wordpress.android.fluxc.network.discovery.SelfHostedEndpointFinder.verifyOrDiscoverXMLRPCEndpoint(SelfHostedEndpointFinder.java:129)
       at org.wordpress.android.fluxc.network.discovery.SelfHostedEndpointFinder.access$100(SelfHostedEndpointFinder.java:40)
       at org.wordpress.android.fluxc.network.discovery.SelfHostedEndpointFinder$1.run(SelfHostedEndpointFinder.java:108)
       at java.lang.Thread.run(Thread.java:818)

```

**Comment by mzorz** [comment]
Also:
```Caused by android.icu.text.StringPrepParseException: The labels in the input are too long. Length > 63.```

**Comment by tonyr59h** [comment]
Closed via #5840
